.github/workflows/docker-build-and-push.yml

```diff
@@ -47,6 +47,16 @@ on:
         required: false
         default: 'https://dev.api.telescope.cdot.systems/v1/status'
         type: string
+      supabase_url:
+        description: 'The Supabase URL'
+        required: false
+        default: 'http://kong:8000'
```

Comment:
This seems wrong.  Our front-end won't be able to access `http://kong`, since it's internal to the Docker network (i.e., only other docker containers running in the same private network space can talk to each other using `http://<service-name>`).  Likely you want `https://dev.telescope.cdot.systems/v1/supabase`, which is the Traefik routed URL that gets you to Kong outside Docker.